packages/client/modules/userDashboard/components/OrgTeamMembers/OrgTeamMembers.tsx

```diff
@@ -88,7 +88,7 @@ export const OrgTeamMembers = (props: Props) => {
         <div className='bg-slate-100 px-4 py-2'>
           <div className='flex w-full justify-between'>
             <div className='flex items-center font-bold'>
-              {teamMembers.length} {plural(teamMembers.length, 'User')}
+              {teamMembers?.length ?? 0} {plural(teamMembers.length, 'User')}
```

Comment:
-1 `teamMembers` should not be undefined, but in any case, it would still fail on the same line when evaluating `plural(teamMembers.length, 'User')` and if that's fixed, then on line 95.

This is really odd, because I think we should have returned in l55. This is the corresponding server error: https://parabol.sentry.io/issues/6565182256/?query=team%20not%20found&referrer=issue-stream&stream_index=0 coming from https://github.com/ParabolInc/parabol/blob/9894b858b4398594508181a9ab1b68cdd27d96f8/packages/server/graphql/public/types/User.ts#L286

Reply:
I think what could have happened was that we returned `null` for the team, but they already had the team with partial data in the client cache. So I guess this could only happen if the permissions for them just changed or something. So maybe change l55 to also check for `teamMembers`

Reply:
Thanks for the insights! I've found the root cause but it's a slightly bigger issue 😂. I'll try to push a fix next week.